### PR TITLE
[MB-1970] Update to Android UA SDK 8.2.2

### PR DESCRIPTION
**Pull request by rlepinski**
Updates to use Android SDK 8.2.2

Changes:
- Update dependency versions
- Bindings for SDK 8.2.2
- Add `Class` to any class that conflicts with the namespace (AnalyticsClass, MessageCenterClass, etc..)

### CODE REVIEW

**Comment by rlepinski** [comment]
Looks like the latest Xamarin update fixed the crash - https://releases.xamarin.com/stable-release-cycle-8-service-release-0/

However we need unreleased versions of v4 and GCM:
 - https://www.nuget.org/packages/Xamarin.Android.Support.v4
 - https://www.nuget.org/packages/Xamarin.GooglePlayServices.Gcm

Holding off on the release until then.

File: src/AirshipBindings.Android/Transforms/Metadata.xml [hunk 1]
```diff
@@ -31,17 +40,11 @@
...
-  <!-- Fix name conflict with BaseIntentService.Delegate and System.Delegate --> 
-  <attr path="/api/package[@name='com.urbanairship']/class[@name='BaseIntentService.Delegate']" name="managedName">BaseIntentServiceDelegate</attr>
-
-  <!-- Rename MessageCenter class to avoid conflict with the namespace -->
-  <attr path="/api/package[@name='com.urbanairship.messagecenter']/class[@name='MessageCenter']" name="managedName">MessageCenterManager</attr>
+ 
+  <!-- Fix field names have the same name as the class for the templates-->
+  <attr path="/api/package[@name='com.urbanairship.analytics']/class[@name='RetailEventTemplate']/field[@name='RETAIL_EVENT_TEMPLATE']" name="managedName">TemplateName</attr>
+  <attr path="/api/package[@name='com.urbanairship.analytics']/class[@name='AccountEventTemplate']/field[@name='ACCOUNT_EVENT_TEMPLATE']" name="managedName">TemplateName</attr>
+  <attr path="/api/package[@name='com.urbanairship.analytics']/class[@name='MediaEventTemplate']/field[@name='MEDIA_EVENT_TEMPLATE']" name="managedName">TemplateName</attr>
```
Comment (hcrowell) [inline src/AirshipBindings.Android/Transforms/Metadata.xml]: Should these three managedNames `TemplateName` match their respective class?

**Comment by hcrowell** [comment]
🍊  after reviewing a question/comment above ^

**Comment by madelgi** [comment]
:boat:

**Pull request merged**